public/data/itzwow.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "AMIT KUMAR BERA",
+    "type": "personal",
+    "bio": "Web Developer, Open Source Enthusiast! & Explorer",
+    "avatar": "https://github.com/itzwow.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itzwow",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itzyourab",
+        "icon": "twitter"
+      },{
```

Comment:
```suggestion
      },
      {
```

Reply:
Github-actions will handle this once the PR is merged so dw about it @syedsohan